apps/www/app/api-v2/ticket-og/route.tsx

```diff
@@ -501,6 +501,7 @@ export async function GET(req: Request, res: Response) {
 
     return await fetch(`${STORAGE_URL}/og/${ticketType}/${username}.png?t=${NEW_TIMESTAMP}`)
   } catch (error: any) {
+    Sentry.captureException(error)
```

Comment:
Since this one is an edge runtime function, it needs to manually flush Sentry events so that they all get sent before the process gets closed: see [related docs PR](https://github.com/supabase/supabase/pull/36377)

```suggestion
    Sentry.captureException(error)
    await Sentry.flush(2000)
```